src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -11,54 +11,106 @@
  * Copyright (c) 2012-2016, Citus Data, Inc.
  */
 
-#include "postgres.h"
+#include "postgres.h" /* IWYU pragma: keep */
```

Comment:
I know, I know! That said, if I'm relying on a data type in a header and don't include it myself, someone's going to deal with the breakage down the line when the transitive include is removed!

`#include`s are there for the machine; let the machine handle it, I say.